### PR TITLE
test: added 2-bit ints to TLPwB tests (related to #6158)

### PR DESCRIPTION
**Pull request by xdesou01**

### CODE REVIEW

File: main/test/flix/Test.Type.Bool.flix [hunk 1]
```diff
@@ -155,6 +155,158 @@ mod Test.Type.Bool {
...
+
+    @Test
+    def testIncrement07(): Bool = isZero(increment(increment(increment(increment(mkZero())))))
+
+    @Test
+    def testIncrement08(): Bool = {
+      let x = increment(increment(increment(mkZero())));
+      let y = increment(increment(increment(x)));
+      let z = increment(increment(increment(y)));
+      isThree(x) and isTwo(y) and isOne(z)
```
[the remark this answers is not in the document]
Comment (xdesou01) [reply main/test/flix/Test.Type.Bool.flix:308]: The goal was just doing a myriad of increments by storing partial results in values, but I guess it's fine to just test the last one. I just figured I might as well assert the other two, but it's not necessary

[the remark this answers is not in the document]
Comment (xdesou01) [reply main/test/flix/Test.Type.Bool.flix:308]: Also, do you think `isOne`, etc, should be renamed to `assertOne,` etc, or does it not matter much?

[the remark this answers is not in the document]
Comment (magnus-madsen) [reply main/test/flix/Test.Type.Bool.flix:308]: I would drop this one.

**Review by magnus-madsen — APPROVED** [review]

**Comment by magnus-madsen** [comment]
Thanks!

**Pull request merged**